pages/osx/GetFileInfo.md

```diff
@@ -0,0 +1,19 @@
+# GetFileInfo
+
+> Get information about a file in an HFS+ directory.
+
+- See date and time of file creation:
+
+`GetFileInfo -d {{filename}}`
+
+- See date and time of latest file modification:
+
+`GetFileInfo -m {{filename}}`
+
+- See the creator of the file:
+
+`GetFileInfo -c {{filename}}`
+
+- See all infomation about the file:
```

Comment:
Perhaps this will read better?
```suggestion
- Display information about the file:
```

Also, perhaps this should be the first example, not the last?